content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -361,3 +361,11 @@ We also recommend restarting any components (e.g. `kube-scheduler`,
 stale data. Note that in practice, the restore takes a bit of time.  During the
 restoration, critical components will lose leader lock and restart themselves.
 {{< /note >}}
+
+## Upgrading etcd clusters
+
+{{< note >}}
+Before you start an upgrade, please back up your Etcd cluster first,
+check the [etcd upgrades](https://etcd.io/docs/latest/upgrades/) docs for the version 
+you actually intend to upgrade and read through the guide to prepare.
+{{< /note >}}
```

Comment:
This instruction doesn't deserve a 'note' shortcode, IMHO.

If there is a source of etcd versions that are known to be working, we'd better provide a link to it.

Reply:
I don't have a good opinion. could you please give some suggestions? cc @sftim

Reply:
I'm happy to have a note shortcode here. Backups are that important.

If the Kubernetes project publishes a list of compatible etcd versions, it may well do but I don't know where that information is collated. SIG API Machinery would own that information, I think.

I'm happy to see this merge as-is and then have a list of compatible etcd versions arrive in a separate PR.